### PR TITLE
added my name to contributor list

### PR DESCRIPTION
**Pull request by ChrisPfeuffer**
I've added my name to the list. @Dhaval1403

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -343,9 +342,7 @@
 
 -[@hashcodepk](https://github.com/Hashcodepk)
 
--[@uniqueayo1988](https://github.com/uniqueayo1988)
--[@kumarnitinsingh](https://github.com/kumarnitinsingh/)
--[@krishnanath](https://github.com/krishnanath)
+-[@uniqueayo1988](https://github.com/uniqueayo1988) -[@kumarnitinsingh](https://github.com/kumarnitinsingh/) -[@krishnanath](https://github.com/krishnanath)
```
Comment (MattCSmith) [inline CONTRIBUTORS.md:345]: Bunching other students up like this is not ideal. 
Any chance you could correct this?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -11,9 +12,7 @@
 
 -[@varunpusarla](https://github.com/varunpusarla)
 
-
--[@seakun](https://github.com/seakun)
--[@swatiidixit](https://github.com/swatiidixit)
+-[@seakun](https://github.com/seakun) -[@swatiidixit](https://github.com/swatiidixit)
```
Comment (MattCSmith) [inline CONTRIBUTORS.md:15]: Bunching other students up like this is not ideal. 
Any chance you could correct this?

Comment (sophiabrandt) [reply CONTRIBUTORS.md:15]: I'm sorry,
it looks like the last merge from another commit merged the bunched up code. That's probably my fault.
Perhaps @ChrisPfeuffer can remedy that in his next pull request?

**Comment by MattCSmith** [comment]
Closing this PR as @ChrisPfeuffer made another PR

**Pull request closed without merging**